lib/time_difference.rb

```diff
@@ -56,9 +56,11 @@ def in_general
     end]
   end
 
-  def humanize
+  def humanize(only_parts = TIME_COMPONENTS)
```

Comment:
I would use a named attribute with a default value here.
```ruby
def humanize(only: TIME_COMPONENTS)
...
```

this would make for an easier more trivial use of the method.